pkg/filters/custom_filter.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package filters
+
+import (
+	"encoding/binary"
+	"math"
+	"math/big"
```

Comment:
This can be done without using `big.Int`... using `big.Int` in hot-path is complicated.